CHANGELOG.md

```diff
@@ -11,6 +11,16 @@ the code was deployed.
 
 ## [Unreleased]
 
+### Added
+
+- `number_of_alerts` integer column to the sessions table (CU-860r8k57h).
+- Chatbot flow to reset a sensor if the `number_of_alerts` in a session exceeds the threshold `SESSION_NUMBER_OF_ALERTS_TO_ACCEPT_RESET_REQUEST` (CU-860r8k57h).
```

Comment:
Just checking. The Responders will be able to reset it if it's >= `SESSION_NUMBER_OF_ALERTS_TO_ACCEPT_RESET_REQUEST`, right? When I read "exceeds", it makes me think that it must be strictly greater than. What do you think?

Reply:
Ahhhh, good catch! I'll reword this to 'meets or exceeds'.